proto/Makefile

```diff
@@ -16,7 +16,7 @@
 # Author: Spencer Kimball (spencer.kimball@gmail.com)
 # Author: Andrew Bonventre (andybons@gmail.com)
 
-PROTOC      := ../_vendor/usr/bin/protoc
+PROTOC      := protoc
```

Comment:
It was straightforward to build protoc using the same cgo approach. See `github.com/cockroachdb/c-protobuf/cmd/protoc`.